src/components/GeoMap.js

```diff
@@ -434,32 +434,43 @@ export default class GeoMap {
       let minValue = attrValue.min;
       let maxValue = attrValue.max;
 
-      if (attrValue.min === 'minValue') {
-        minValue = scaleDomain[0];
-      }
-      if (attrValue.max === 'maxValue') {
-        maxValue = scaleDomain[1];
+
+      // check for negative color declorations
+      var useNegative = false;
+      if ((attrValue.maxNegative && !attrValue.minNegative) ||
+          (!attrValue.maxNegative && attrValue.minNegative)) {
+        throw new Error(`MapTable: maxNegative or minNegative undefined. Please declare both.`);
+      } else if (attrValue.maxNegative && attrValue.minNegative){
+        useNegative = true;
```

Comment:
You can remove the declaration on L439 and use:
`const useNegative = (attrValue.maxNegative && attrValue.minNegative);`